geowebcache/web/src/test/java/org/geowebcache/jetty/JettyRule.java

```diff
@@ -1,31 +1,31 @@
 package org.geowebcache.jetty;
 
+import org.eclipse.jetty.server.*;
+import org.eclipse.jetty.util.thread.QueuedThreadPool;
+import org.eclipse.jetty.webapp.WebAppContext;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
 import java.io.File;
-import java.net.BindException;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.util.Objects;
 import java.util.function.Consumer;
 
-import org.junit.rules.TemporaryFolder;
-import org.junit.runner.Description;
-import org.junit.runners.model.Statement;
-import org.mortbay.jetty.Connector;
-import org.mortbay.jetty.Server;
-import org.mortbay.jetty.bio.SocketConnector;
-import org.mortbay.jetty.webapp.WebAppContext;
-import org.mortbay.thread.QueuedThreadPool;
+//import org.eclipse.jetty.bio.SocketConnector;
+//import org.eclipse.thread.QueuedThreadPool;
 
 /**
  * Rule that runs a live GWC instance inside of Jetty for integration tests
- * 
+ *
  * @author Kevin Smith, Boundless
  *
  */
 public class JettyRule extends org.junit.rules.ExternalResource {
-    
+
     Integer port = null;
-    
+
```

Comment:
Yes, I'm not sure how they got in there.  Might be an artifact of copy/paste between applications.